packages/block-library/src/author-name/block.json

```diff
@@ -0,0 +1,42 @@
+{
+	"apiVersion": 2,
+	"name": "core/author-name",
```

Comment:
I think it should be `core/post-author-name` and prefixed with Post in other places. This block depends on the context that passes the post id. It will help to differentiate it with a very similar block for comments (Comment Author Name) that depends on the comment id passed from the context.